packages/node/test/fixtures/operation/requests.ts

```diff
@@ -1,4 +1,17 @@
-import { FullRequest, Withdrawal } from '@airnode/operation';
+import { RegularRequest, FullRequest, Withdrawal } from '@airnode/operation';
+
+export function buildRegularRequest(overrides?: Partial<RegularRequest>): RegularRequest {
```

Comment:
I really thought that there was a fixture function for regular requests already. Guess not 🤷🏻‍♂️